resin_release_tool/releaser.py

```diff
@@ -21,6 +24,68 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
+        all_apps = self.models.application.get_all()
+        if len(next((app for app in all_apps if app['id'] == int(self.app_id)),{})) != 0: #checks if the app exists
```

Comment:
This line is a bit too long/dense/hard to understand, maybe move the main part of the expression to a different line

Reply:
closing the conversation since it has been refactored.